### PR TITLE
electrum: Update to 2.8.3

### PR DESCRIPTION
**Pull request by Sir-Boops**
This also adds two new python packages as electrum now depends on them

### CODE REVIEW

**Comment by Vaelatern** [comment]
Python modules look wrong, lacking a version 2 and ver. 3 package. See other python packages to be a general guide.

**Comment by Sir-Boops** [comment]
Added the python3 version of the two new python packages

**Comment by Vaelatern** [comment]
All transitional dummy packages are no longer relevant, since new packages don't need to migrate from old number schemes

**Comment by Vaelatern** [comment]
Descriptions also need to follow the model of other python packages

File: srcpkgs/python-pysocks/template [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Template file for 'python-pysocks'
+pkgname=python-pysocks
+version=1.6.7
+revision=1
+wrksrc=PySocks-${version}
+noarch=yes
+build_style=python-module
+depends="python"
+hostmakedepends="python-setuptools python3-setuptools"
+short_desc="PySocks lets you send traffic through SOCKS and HTTP proxy servers"
```
Comment (Vaelatern) [inline srcpkgs/python-pysocks/template]: How this would look:
`[-] python-pysocks-1.6.7_1       Pysocks ....`
don't repeat the package name!

**Comment by Sir-Boops** [comment]
Updated the descriptions and removed the python3.4 version of the packages

**Comment by Gottox** [comment]
@Vaelatern okay from you?

**Pull request merged**